doc/docs.md

```diff
@@ -2958,8 +2972,7 @@ The strings don't escape `draw_text`, so they are cleaned up when
 the function exits.
 
 In fact, the first two calls won't result in any allocations at all.
-These two strings are small,
-V will use a preallocated buffer for them.
+These two strings are small, so V will use a preallocated buffer for them.
```

Comment:
This is not true anymore; the preallocated buffer is behind an experimental flag `-prealloc` now, and may not stay.
I think this paragraph should be deleted.
@medvednikov what do you think?

Reply:
Updated to make note of the flag. Can revise in the future.